controllers/extensionRequests.js

```diff
@@ -202,12 +202,23 @@ const getSelfExtensionRequests = async (req, res) => {
  * @param res {Object} - Express response object
  */
 const updateExtensionRequest = async (req, res) => {
+  const { dev } = req.query;
+  const isDev = dev === "true";
   try {
     const extensionRequest = await extensionRequestsQuery.fetchExtensionRequest(req.params.id);
     if (!extensionRequest.extensionRequestData) {
       return res.boom.notFound("Extension Request not found");
     }
 
+    if (
+      isDev &&
+      !req.userData?.roles.super_user &&
+      (extensionRequest.extensionRequestData.status === EXTENSION_REQUEST_STATUS.APPROVED ||
+        extensionRequest.extensionRequestData.status === EXTENSION_REQUEST_STATUS.DENIED)
+    ) {
+      return res.boom.badRequest("Extension Request cannot be updated");
+    }
```

Comment:
`Extension Request cannot be updated` is a little bit generic. It would be beneficial to provide more context on why the request cannot be updated

Reply:
Got it. The message is updated to `Only pending extension request can be updated`. Please let me know if this is better.